otx/algorithms/classification/adapters/mmcls/task.py

```diff
@@ -543,17 +544,18 @@ def hook(module, inp, outp):  # pylint: disable=unused-argument
 
         return eval_predictions, saliency_maps
 
-    def _export_model(self, precision, dump_features):
+    def _export_model(self, precision: ModelPrecision, export_format: ExportType, dump_features: bool):
         self._init_task(export=True)
 
         cfg = self.configure(False, "test", None)
 
         self._precision[0] = precision
         export_options: Dict[str, Any] = {}
         export_options["deploy_cfg"] = self._init_deploy_cfg(cfg)
-        if export_options.get("precision", None) is None:
-            assert len(self._precision) == 1
-            export_options["precision"] = str(self._precision[0])
+
+        assert len(self._precision) == 1
+        export_options["precision"] = str(self._precision[0])
```

Comment:
Minor comments, how about using `precision` instead of `self._precision[0]` for this line and [line#570](https://github.com/openvinotoolkit/training_extensions/pull/2155/files#diff-5796e4e6b54d4951de4e661020dc39b486a6ff2e709a4a02df07c43d223fd6e6R570)?
First one is more readable to me.

Reply:
Changed